src/main/scripts/ncm-query

```diff
@@ -133,7 +133,9 @@ toolsuite.
 
 package query;
 
+pop @INC if $INC[-1] eq '.';
```

Comment:
@ttyS4 wrap this in `BEGIN{}` like the other patches

Reply:
Good point I have just updated as that.